release-notes/opensearch.job-scheduler.release-notes-1.0.0.0-beta1.md

```diff
@@ -16,7 +16,7 @@ Compatible with OpenSearch 1.0.0
 ### Infrastructure
 * Build and publish to maven local, enable CI ([#8](https://github.com/opensearch-project/job-scheduler/pull/8))
 * Reset version to 1.0. ([#9](https://github.com/opensearch-project/job-scheduler/pull/9))
-* Build against OpenSearch 1.0.0-alpha1. ([#11](https://github.com/opensearch-project/job-scheduler/pull/11))
+* Build against OpenSearch 1.0.0-rc1. ([#11](https://github.com/opensearch-project/job-scheduler/pull/11))
```

Comment:
Looks like this was accidentally included when replacing. This should stay as alpha1